roles/dns-config/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: assert that variables are defined
+  fail:
+    msg: "Variable '{{ item }}' is not defined"
+  when: item not in vars
+  with_items:
+    - dns_config_servers
```

Comment:
This doesn't match the var in `roles/dns-config/tasks/main.yml`. It's also ambiguous whether we're configuring nameservers or search domains, I would think we want to have vars for both.

Reply:
I see what you're doing now, which is the same with the KRB stuff. I'm not sure about this, because it requires a user to set config vs. having sane defaults out of the box. Can we just make no changes to the file if the variable is not set?